### ISSUE TITLE
tapochek: torrent block carried no usable fields

### ISSUE THREAD

**Issue by odcold** [artyomsv/marauder#186]
### Marauder version

v1.19.10

### How are you running Marauder?

docker compose (production stack)

### What happened?

<img width="1521" height="1116" alt="Image" src="https://github.com/user-attachments/assets/adacdf80-20aa-4941-8ac5-075804b5aebd" />

### Steps to reproduce

add topic, eg https://tapochek.net/viewtopic.php?t=288010
experience the error above there is also an issues with posters 

### What did you expect to happen?

.

### Backend logs

```shell
docker logs marauder-backend-1 2>&1 | tail -200 | grep tapochek
{"level":"warn","service":"marauder-backend","component":"scheduler","worker":3,"topic_id":"8fc3790b-6ce6-4fea-af3c-c62680c9a29f","tracker":"tapochek","url":"https://tapochek.net/viewtopic.php?t=288010","error":"tapochek: torrent block carried no usable fields","time":"2026-09-20T23:27:16.010296743Z","message":"check failed"}
```

**Comment by artyomsv** [comment]
@odcold Thanks for the report — there are two separate things here.

**Posters: confirmed bug, fixed.** Tapochek serves two page templates. TV-series  topics put the cover in an `<img class="poster">` tag, and app only read the  `<var class="postImgAligned">` form used by film and game topics. All three of  your topics now resolve their cover.

**`torrent block carried no usable fields`: I could not reproduce it.** All  three of your topics check cleanly here on the same version. The error means the  torrent table was on the page, but the `download.php` link was missing from it.  Your titles resolved, so your session is alive — this looks like your Tapochek  account not being permitted to download those releases (ratio, rank, or a daily  download cap). They are large: 13 GB, 16 GB and 7.5 GB.

Could you open https://tapochek.net/viewtopic.php?t=288010 in a browser, signed  in with the same account, and tell me whether the green download button is there?

Either way the message was wrong to blame the parser, so it now reads:  `no download link in the torrent table — this account may not be allowed to  download this release (ratio, rank or daily download limit)`, and the generic  case names the field it lost.

**Comment by odcold** [comment]
@artyomsv 
it's not an account related issue, it has 168.37 ratio and i'm not experiencing any problems while using browser for torrent file downloads or posters preview. Also I can provide an invite for testing purposes

**Comment by artyomsv** [comment]
@odcold try to use updated v1.20.1 version to see what would be the error message. After that if it will not be clear I will add additional logging to get more info from the processing

**Comment by odcold** [comment]
@artyomsv 

> backend-1   | {"level":"warn","service":"marauder-backend","component":"scheduler","worker":0,"topic_id":"8fc3790b-6ce6-4fea-af3c-c62680c9a29f","tracker":"tapochek","url":"https://tapochek.net/viewtopic.php?t=288010","error":"tapochek: torrent block carried no usable fields (missing: name)","time":"2026-09-21T13:18:58.541751567Z","message":"check failed"}

so, now we have

"missing name" in error details

**Comment by artyomsv** [comment]
@odcold That narrowed it right down — thank you.

`(missing: name)` means one cell only: the table's top bar with the `.torrent` file name. Your size, date and download link all parse fine, so the table is intact. On my account that cell is:
````html
<th colspan="3" class="genmed">…[tapochek.net].torrent</th>
````
Plain text inside a `<th class="genmed">`. On yours it is not, and that is the  whole bug.

I also ruled out the theme: I fetched that exact topic in system, light and dark mode with a live session and the cell is byte-identical in all three.

So it looks like a per-account profile setting - a different board style or a different board language - renaming or re-wrapping that cell. To confirm, could you send me the raw HTML of just that table?

    1. Open https://tapochek.net/viewtopic.php?t=288010
    2. Press Ctrl+U to view the page source
    3. Ctrl+F for: attach bordered
    4. Copy that <table …> … </table> block and paste it here

**Comment by odcold** [comment]
@artyomsv 

```
<table class="attach bordered med">
	<tr class="row3">
		<th colspan="3" class="seedmed">Стюарт Блум не смог спасти вселенную Stuart Fails to Save the Universe Сезон 1 Серии 1-8 из 10 [WEB-DL 1080p] [tapochek.net].torrent</th>
	</tr>
    

		<tr class="row4">
	    <th colspan="3" class="row7 gold-header torrent-type-header torrent-type-header--gold"><img src="images/tor_gold.gif" width="16" height="15" title="Золото" />&nbsp;ЗОЛОТАЯ РАЗДАЧА! СКАЧАННОЕ НЕ ЗАСЧИТЫВАЕТСЯ!&nbsp;<img src="images/tor_gold.gif" width="16" height="15" title="Золото" />&nbsp;</th>
	</tr>
			<tr class="row1">
		<td width="15%">Трекер:</td>
		<td width="70%">
						Зарегистрирован &nbsp;
						[ <span title="4 дня">17-09-2026 00:21</span> ]
		</td>
		<td width="15%" rowspan="7" class="tCenter pad_6">
						<a href="download.php?id=188304" class="seedmed">
			<p><span class="download-torrent-btn" title="Скачать торрент-файл">&#8681; Скачать &#8681;</span></p></a>
			<p class="small torrent-size-info">Размер .torrent файла 131&nbsp;KB</p>
						<br /><p class="small"><input type="button" value="Список файлов" id="tor-filelist-btn" onclick="show_filelist();return false;"></p>
		
                
        </td>
	</tr>

	

<tr class="row1">
<td>Статус:</td>
<td style="padding: 6px 4px;">

<script type="text/javascript">$('#tor-288010').html( $('#tor-status-txt').html() );</script>
<b>
<span  id="music-288010-icon" style="color:red">
</span>
<span  id="tor-288010-icon" class="tor-icon tor_status_txt">
<span style="color: green;">&radic;</span></span> <span id="tor-288010-text"> проверено</span>  
</b></a>
&nbsp;&middot;&nbsp; <span id="tor-288010-user">(by <a href=./profile.php?mode=viewprofile&u=522498>Fire</a> in 17-09-2026 00:22 &middot; <i>4 дня</i> назад)</span>
</span>
<br /><span  id="music-288010-text" style="color:red"></span> 
		
</td>
</tr>
	<tr class="row1">
		<td>Скачан:</td>
		<td><span title="Раздача полностью скачана 38 раз">38 раз</span></td>
	</tr>
	<tr class="row1">
		<td>Размер:</td>
		<td>13.03&nbsp;GB</td>
	</tr>
	<tr class="row1">
		<td>Поблагодарили:</td>
		<td>
			<span class="vMiddle" style="display: flex; align-items: center;">
				<script language="JavaScript" type="text/javascript" src="./misc/js/ajax.js"></script>
				<span id="VB188304"> <a href="#" class="thanks-btn" title="Поблагодарить автора раздачи" onclick="this.onclick=null; this.className+=' thanks-btn--disabled'; ajax_do('thanks.php?mode=thank&amp;a=188304'); return false;">Спасибо</a></span> &nbsp;&nbsp;<span id="VT188304">25</span> &nbsp; (<span id="VL188304"><a href="#torrent" onClick="ajax_do('thanks.php?mode=list&a=188304');">список</a></span>)			</span>
		</td>
	</tr>
	
	</tr>
		<tr class="row3 tCenter">
		<td colspan="3">
				&nbsp;
				</td>
	</tr>
<tr>
	<td width="100%" colspan="3" class="attachheader" align="center"><b><div id="vid_vpaut_div" style=" display:inline-block;width:600px;height:320px" vid_vpaut_pl="9719"></div><script type="text/javascript" src="https://videoroll.net/js/vid_vpaut_script.js" async></script></td>
</tr>
</table>
```

**Comment by odcold** [comment]
@artyomsv thanks. Everything looks good so far
